proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,531 @@
+   maintainers should happen in a timely manner, even (especially!) if that’s
+   saying no. An initial response to open PRs should occur within [TBD] business
+   days.
+1. A contribution should not be left abandoned. The maintainers are responsible
+   see it through to a conclusion.
+1. Contributors should know exactly what to expect when interacting with the
+   maintainers. The current status should be clearly indicated and the next
+   steps well defined.
+
+## Specification
+
+ ![Figure A: Flowchart of contribution guide review
+process.](contrib-guide-high-level.png "Figure A: Flowchart of contribution
+guide review process.")
+
+**Figure A: Flowchart of contribution guide review process.**
+
+We propose to formalize the existing intake, implementation, and review stages
+to assure they are applied systematically and transparently. In addition, we
+will augment the current process with new stages to clarify what issues are
+available for contribution and what stage they are in. Figure A demonstrates how
+a request and contribution will proceed through the improved process.
+
+We will also publish an improved contributing guide to document the review
+process, acceptance criteria, and expectations for communication. We will use
+existing issue labeling in conjunction with a GitHub Project to provide
+transparent status to the community to indicate what is available for
+contribution and to track the progress of a contribution.
+
+The following sections detail the proposed improvements to the existing process.
+
+### Intake
+
+We will modify the the intake process to set contribution expectations for the
+community. A contribution is initiated by opening a GitHub issue. The GitHub
+issue captures information necessary to make the contribution, such as:
+
+1. A description of the requested change.
+1. The number of users in the community that request the change, measured
+   through GitHub :+1: reactions.
+1. Design discussions, including use cases, edge cases, and alternative
+   solutions.
+
+Contributions made without a GitHub issue will be subject to closure by the
+maintainers.
+
+Maintainers will regularly review open GitHub issues to determine if they are
+ready for community contribution. The decision to label an issue as ready for
+community contribution is ultimately up to the maintainers. Depending on the
+issue type (bug or feature request) and area (code or documentation), the
+maintainers will use a set of criteria to decide if the issue is ready for
+contribution. This may include:
+
+1. The change is feasible for a community member to contribute. See [the
+   appendix](#changes-not-available-for-contribution) for details.
+1. Feature requests must have at least 10 :+1: reactions.
+1. The maintainers agree with the implementation plan.
+
+If the criteria are not satisfied, the maintainers will comment on the GitHub
+issue to request more information and to update the current status or next
```

Comment:
I'm not sure if this is specific enough. In general, if an issue does not meet our criteria, a maintainer should explicitly enumerate the criteria the issue needs to meet and what steps the community can do in in order to meet it. The problem the way it is phrased right now is there is no explicit requirement for stating what is the full list of criteria that is left, which could:

1. Make it more difficult for the community to understand what is left to reach ready for contribution if the criteria is not exactly spelled out.
2. Harder for maintainers to go back and refresh themselves on where the issue stood and what criteria they originally had determined.